encoding/values_test.go

```diff
@@ -0,0 +1,12 @@
+package encoding_test
+
+import (
+	"testing"
+	"unsafe"
+
+	"github.com/segmentio/parquet-go/encoding"
+)
+
+func TestValuesSize(t *testing.T) {
+	t.Log(unsafe.Sizeof(encoding.Values{}))
```

Comment:
Make an assertion here?

Reply:
I don't think we can, the size depends on the size of `int` which is "at least 32 bits" but is usually 64 bits on AMD64/ARM64.